reconcile/cli.py

```diff
@@ -2628,10 +2628,10 @@ def prometheus_rules_tester(ctx, thread_pool_size, cluster_name):
 
 @integration.command(short_help="Tests templating of resources.")
 @click.pass_context
-def template_tester(ctx):
-    import reconcile.template_tester
+def resource_template_tester(ctx):
```

Comment:
Rename here is a breaking change, need to have additional promotion notes.

Reply:
For fedramp? Will create one